friends/hello/__init__.py

```diff
@@ -0,0 +1,32 @@
+import check50.check50
+import check50.check50.c
+
+@check50.check()
+def exists():
+    """hello.c exists"""
+    check50.exists("hello.c")
+
+@check50.check(exists)
+def compiles():
+    """hello.c compiles"""
+    check50.c.compile("hello.c", lcs50=True)
+
+@check50.check(compiles)
+def test_hello_world():
+    """prints "Hello, world!" if argv[1] is "world" """
+    check50.run("./hello world").stdout("Hello, world!\n", "Hello, world!").exit(0)
+
+@check50.check(compiles)
+def test_hello_elphie():
+    """prints "Hello, elphie!" if argv[1] is "elphie" """
+    check50.run("./hello elphie").stdout("Hello, elphie!\n", "Hello, elphie!").exit(0)
```

Comment:
Pedagogically, any reason to hide that we are looking for a newline @Erin-c? Currently if a student writes a program that prints "Hello, elphie!" sans newline, the check will fail, and they will see `expected "Hello, elphie!, but got "Hello, elphie!"` which is pretty nonsensical. At the very least, `.stdout("Hello, elphie!\n")` would result in the message `expected "Hello, elphie!\n", but got "Hello, elphie!"` which is slightly better. For even more helpful output, perhaps the [hello](https://github.com/ehhong/problems/blob/2018/ap/hello/__init__.py) check would be good inspiration here?

Reply:
we can include `\n` in the error message